paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -695,10 +695,16 @@ def valid_transitions(self) -> Iterator[state_machine.TransitionDefinition]:
     def disable_auto_rollbacks(self):
         self.cancel_auto_rollback_countdown()
         self.auto_rollback = False
+        self.update_slack_status(
+            f"Automatic rollback disabled for this deploy. To disable this permanently for this step, `edit deploy.yaml`"
```

Comment:
> To disable this permanently for this step, edit `deploy.yaml`
instead of 
> To disable this permanently for this step, `edit deploy.yaml`

Also maybe worth mentioning what config key to use?